### PR TITLE
Allow complex root password

### PR DESCRIPTION
**Pull request by lewismarshall**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ curl -fail http://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem -o /
 # Allow for no passwords when running mysql as root...
 echo "[client]
 user=root
-password=$(cat ${ROOT_PASS_SECRET})
+password='$(cat ${ROOT_PASS_SECRET})'
```
Comment (vaijab) [inline docker-entrypoint.sh:56]: That will never be able to read `${ROOT_PASS_SECRET}` because single quotes.

Comment (lewismarshall) [inline docker-entrypoint.sh:56]: No it won't - it's not a bash file - it's a .my.cnf file and tested it on the actual container...

Comment (lewismarshall) [inline docker-entrypoint.sh:56]: @vaijab 

```
[root@mysqlmaint /]# grep 'my.cnf' docker-entrypoint.sh -B 3
echo "[client]
user=root
password='$(cat ${ROOT_PASS_SECRET})'
">~/.my.cnf
[root@mysqlmaint /]# ./docker-entrypoint.sh
Downloading CA cert for mysql access
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 18528  100 18528    0     0  56383      0 --:--:-- --:--:-- --:--:-- 56487
Check default PW access...
Update / create any database users...
[root@mysqlmaint /]#
```

**Comment by vaijab** [comment]
LGTM

**Pull request merged**